modelcontextprotocol/server.py

```diff
@@ -10,7 +11,7 @@
 )
 from pyatlan.model.lineage import LineageDirection
 
-mcp = FastMCP("Atlan MCP", dependencies=["pyatlan"])
+mcp = FastMCP("Atlan MCP Server", dependencies=["pyatlan"])
```

Comment:
lets add `fastmcp` as well to the dependencies, in case if someone tries out the command
```suggestion
mcp = FastMCP("Atlan MCP Server", dependencies=["pyatlan", "fastmcp"])
```

```
uv run mcp install server.py -f .env
```

the server fails to connect, as the `fastmcp` deps is missing.

Reply:
this helps any dev, to just add the mcp directly via terminal.

Reply:
Done